infrastructure/state.tf

```diff
@@ -11,7 +11,7 @@ terraform {
     }
     azuread = {
       source  = "hashicorp/azuread"
-      version = "1.6.0"
+      version = "~> 2.0.0"
```

Comment:
best to fix the version to latest working

Reply:
Not sure what you mean. The provider wasn't explicitly in the state.tf until now so I added it to get round the terraform show error we've been seeing recently. Assuming it needs 2.0.0 going by the error in the latest pipeline run. 

`Could not retrieve the list of available versions for provider
hashicorp/azuread: no available releases match the given constraints 1.6.0,>= 2.0.0`

Reply:
be explicit:
```suggestion
      version = "2.5.0"
```

Reply:
Updated